python/gps/algorithm/policy/policy_prior_gmm.py

```diff
@@ -65,6 +65,8 @@ def update(self, samples, policy_opt, all_samples, retrain=True):
             self.X = self.X[start:, :, :]
             self.obs = self.obs[start:, :, :]
             # Evaluate policy at samples to get mean policy action.
+            if m is not None:
+                Upol = policy_opt.prob(self.obs.copy(), m)[0]
```

Comment:
[If you decide to delete it, please also delete the argument m to this function.